packages/fxa-shared/metrics/amplitude-event.1.schema.json

```diff
@@ -296,9 +299,16 @@
         }
       },
       "then": {
+        "required": ["user_id"],
```

Comment:
Refactoring the tests as a follow-up from https://github.com/mozilla/fxa/pull/15190 (FXA-6953/FXA-6954) actually discovered one validation requirement that we missed for plan change events -- requiring the `user_id`. One of the reasons we write tests!